test/units/plugins/strategy/test_strategy_base.py

```diff
@@ -418,7 +418,7 @@ def _has_dead_workers():
         # self.assertEqual(strategy_base._pending_results, 1)
 
         # queue_items.append(('bad'))
-        # self.assertRaises(AnsibleError, strategy_base._process_pending_results, iterator=mock_iterator)
+        # self.assertRaises(strategy_base._process_pending_results, iterator=mock_iterator)
```

Comment:
I don't think this line should be modified.

Reply:
@felixfontein , good job! bash automation affected.. ) I should have checked in detail. Thank you!